javascript/presigned-url-example.ts

```diff
@@ -0,0 +1,140 @@
+import {MomentoSigner, CacheOperation} from '@gomomento/sdk';
+import fetch, {Response} from 'node-fetch';
+
+const signingKey = process.env.MOMENTO_SIGNING_KEY;
+if (!signingKey) {
+  throw new Error('Missing required environment variable MOMENTO_SIGNING_KEY');
+}
+const signingEndpoint = process.env.MOMENTO_ENDPOINT || '';
+if (signingEndpoint === '') {
+  throw new Error('Missing required environment variable MOMENTO_ENDPOINT');
+}
+
+let cacheName: string;
+if (process.env.CACHE_NAME) {
+  cacheName = process.env.CACHE_NAME;
+} else {
+  cacheName = 'default-cache';
+  console.log(
+    `Environment variable CACHE_NAME not set. Using cache name '${cacheName}'.`
+  );
+}
+const cacheKey = 'MyCacheKey';
+const cacheValue = 'MyCacheValue';
+const expiryEpochSeconds = Math.floor(Date.now() / 1000) + 10 * 60; // 10 minutes from now
+const objectTtlSeconds = 180;
+
+const main = async () => {
+  const signer = await MomentoSigner.init(signingKey);
+
+  console.log('\n*** Running presigned url example ***');
```

Comment:
These messages don't really give much indication of what these examples are doing or why a user would want to use them.  I think that's important for an examples repo.  We should try to establish a pattern here to help users understand this stuff and give us some organizational structure going forward.

Some options we could consider:

* create a subdir for each example and put a README in each subdir, explaining what the example does.  
* add a section to the main README that provides the missing context on what these examples do and why.
* Put some documentation links in the console log output here, e.g. `For more info on momento pre-signed URLs, see https://gomomento.com/docs/presigned-urls`.

Reply:
I tried to add some more context via log messages and comments in the code in [1374c99](https://github.com/momentohq/client-sdk-examples/pull/102/commits/1374c9932a233e07c090b5676b67d25ecd552cbd). Unfortunately we don't yet have docs on this :( (but I opened an issue for that).

Here's what output looks like now:
```
$ npm run presigned-url-example

> momento-javascript-example@1.0.0 presigned-url-example
> tsc && node dist/presigned-url-example.js

Creating cache 'MyCache'
[1660248321187] INFO (Momento/49547 on griddle.local): Creating cache: MyCache
Cache 'MyCache' already exists
Checking for signing key in $MOMENTO_SIGNING_KEY
Did not find signing key in environment, creating new signing key

*** Running presigned url example ***
Creating signed url for SET for cache 'MyCache' and key 'MyCacheKey'
Creating signed url for GET for cache 'MyCache' and key 'MyCacheKey'
Created signed urls with claims: exp = 1660248921, cache = MyCache, key = MyCacheKey, ttl seconds (for set) = 180
Setting value for cache 'MyCache' and key 'MyCacheKey' via HTTP with signed URL: 'MyCacheValue'
Getting value for cache 'MyCache' and key 'MyCacheKey' via HTTP with signed URL
Retrieved value with signed URL: "MyCacheValue"
Presigned url example finished successfully!

*** Running signed token example ***
Creating new signed access token for cache 'MyCache'
Created signed access token for cache 'MyCache' with expiry 1660248921
Setting value for cache 'MyCache' and key 'someKey' via HTTP with signed token: 'somedata'
Retrieved value via URL with signed token: "somedata"
Signed token example finished successfully!

success!!
```

Reply:
that is a huge improvement, thanks